### ISSUE TITLE
Great work, could you share some performance comparisons of your implementation?

### ISSUE THREAD

**Issue by PkuRainBow** [bnulihaixia/Deep_metric#2]
It is very good to keep such a repo to summarize all the metric learning related loss functions' implementation. I am wondering whether it is convenient for you to share us the evaluation results to get more insights about whether your implementation is comparable with their orignal implementations.

Besides, you mentioned that your method named below has achieved state-of-the-art performance.

`New Positive Mining Loss based on Fuzzy Clustering`

Have you compared your method with some recent work like HDC[1], BIER[2] and so on.

[1] : Yuan Y, Yang K, Zhang C. Hard-Aware Deeply Cascaded Embedding[J]. arXiv preprint arXiv:1611.05720, 2016. 
[2]: Opitz, Michael, et al. "BIER-Boosting Independent Embeddings Robustly." Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition. 2017.

**Comment by bnuwangxun** [comment]
Sorry for response late, my performance will be updated soon. I was quit busy these days. 
On CUB200, my method is the first one to reach rank-1 higher than 0.5.

**Comment by PkuRainBow** [comment]
@bnuwangxun Please check the current state-of-the-art performance.  As far as I am concerned, the current SOTA could achieve higher than 0.6. Please refer to  https://arxiv.org/abs/1801.04815

**Comment by bnuwangxun** [comment]
the current best results of my method follows, and I am still trying to make my method better now:
<img width="645" alt="2018-01-24 4 42 16" src="https://user-images.githubusercontent.com/22806601/35322604-d94133ec-0125-11e8-9f3c-da4f38368337.png">
<img width="815" alt="2018-01-24 4 42 28" src="https://user-images.githubusercontent.com/22806601/35322611-db2a62c8-0125-11e8-9d19-48e4f460e25e.png">
<img width="705" alt="2018-01-24 4 42 35" src="https://user-images.githubusercontent.com/22806601/35322618-dd6c2ad0-0125-11e8-82d0-db5ecc08d788.png">

**Comment by bnuwangxun** [comment]
Thanks very much for your information. I have a quick look at that paper.  (https://arxiv.org/abs/1801.04815)
The paper has done better performance, however my method is just use one end-to-end network without ensemble learning method.

**Comment by PkuRainBow** [comment]
They are also using one single network indeed with multiple losses. (DSN manner)

**Comment by bnuwangxun** [comment]
Some methods just report their results with test-augment. such as N-pairs.
For fair compare, we use the results from Proxy NCA Loss, which give the results of such methods without test-augment.
Unlike the paper you provide, my work try to propose a new method to train one network by a new loss function. So I just compare my method with methods that doing things in similar ways. 
I will carefully read the papers you mentioned these days, and share my points with you.

**Comment by bnuwangxun** [comment]
Thanks very much, I will read these papers you mentioned.

**Comment by bnuwangxun** [comment]
I have read the papers you mentioned, they have done great works. And I think HDC , BIER , and PCB in Reid are doing deep metric in a way. They pay main attention on refining the architecture of network. in my work, I just modified the triplet loss a little, and then reach the performance comparable with Proxy-NCA.
We do deep metric in an independent way, and I think my loss can be combined with BIER very easily. I will try to do these soon, and share you the results.

**Comment by bnulihaixia** [comment]
@PkuRainBow I have done a new work which reach higher than 60 in rank1 in CUB 200 without data augment. The result  is very stable. 
BUT, unfortunately, in  ICLR-18 someone do the same work as me, for different point of view.

**Comment by PkuRainBow** [comment]
@bnulihaixia If you have interest to intern at MSRA, you could send me your CV to  yuyua@microsoft.com

**Comment by bnulihaixia** [comment]
Thanks!  I will send my CV to you these days.

**Comment by asanakoy** [comment]
@PkuRainBow  Here is the link to review this paper: https://openreview.net/pdf?id=Skf5qiC5KQ
You can ask questions there.

**Comment by bnulihaixia** [comment]
No, it is a old work with  much lower performance. I am doing a new work.  I will share my work with you when it is done.
